android/app/src/main/java/com/chorestory/fragment/ParentQuestsPendingFragment.java

```diff
@@ -0,0 +1,22 @@
+package com.chorestory.fragment;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.chorestory.R;
+
+public class ParentQuestsPendingFragment extends Fragment {
+
+    @Nullable
```

Comment:
Is this necessary if you're specifying which args can be null? can the view be null?

Reply:
idk I just copied Jim's fragment, @jimzhaozhengwang ??

Reply:
> When using javax.annotation.Nullable to mark methods as "potentially returning null value", where to put the @Nullable annotation? On the method, or return type?

Reply:
We can remove all of these

Reply:
I will remove it from the files I modified as well